### PR TITLE
Client profile page

### PR DESCRIPTION
**Pull request by MelindaCardenas**
It's finally here! I'm attaching some photos and a video below to display different screen sizes and it's functionality. (the little gray dot is the cursor, the big gray circle is what shows when someone doesn't have a profile photo)

I'd love any design recommendations since i kind of winged it at certain points, especially with the cropping feature

iphone: 
iPhone_screen

ipad:
iPad_screen

modal:
modal popup

cropper:
cropper

computer:
computer

activities:
activities_computer screen

video : 
[client-profile-page-demo_4MB.mov.zip](https://github.com/keepid/keepid_client/files/5614856/client-profile-page-demo_4MB.mov.zip)

### CODE REVIEW

**Comment by mishyi** [comment]
- More space at the top right above the client profile picture (increase pixel value for padding-top)
- More space between the rows with the Username, Password, Birthdate, etc
- To make the button stand out a bit more, maybe the button can have a purple background with white text instead of a purple outline! When the users hover over the button, the button can then have a purple outline with a white background instead of the shadow/glow

I'll ping you with screenshots as well! :)

File: public/index.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
     <!-- Content Security Policy -->
     <!-- <meta http-equiv="Content-Security-Policy" content="
       default-src 'self' 'unsafe-inline';
-      script-src 'self' https://ajax.cloudflare.com/ https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js https://code.jquery.com/jquery-3.4.1.slim.min.js localhost:* http://127.0.0.1/ http://localhost/ https://js.hsforms.net/ https://forms.hsforms.com/
+      script-src 'self' 'unsafe-inline' https://ajax.cloudflare.com/ https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js https://code.jquery.com/jquery-3.4.1.slim.min.js localhost:* http://127.0.0.1/ http://localhost/ https://js.hsforms.net/ https://forms.hsforms.com/
```
Comment (loafyyy) [inline public/index.html:37]: Why was this change made and why is this section commented out?

File: package.json [hunk 2]
```diff
@@ -9,6 +9,8 @@
     "@nivo/bar": "^0.62.0",
     "@nivo/line": "^0.62.0",
     "@nivo/pie": "^0.62.0",
+    "@sentry/browser": "^5.27.4",
+    "@sentry/tracing": "^5.27.4",
```
Comment (loafyyy) [inline package.json:13]: Can you briefly describe what these packages are and why the need to be added?

File: src/components/AccountSecurity/MyOrganization.tsx [hunk 3]
```diff
@@ -279,9 +279,11 @@ class MyOrganization extends Component<Props, State> {
       }),
     }).then((response) => response.json())
       .then((responseJSON) => {
-        const responseObject = responseJSON;
+        const responseObject = JSON.parse(responseJSON);
         const { status } = responseObject;
+
         if (status === 'SUCCESS') {
+          console.log('it was a success');
```
Comment (loafyyy) [inline src/components/AccountSecurity/MyOrganization.tsx:286]: Can you please remove the consol.log statements?

File: src/components/FindUsername.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```
Comment (loafyyy) [inline src/components/FindUsername.tsx:1]: Why is this a separate component? It looks like it can be combined with ClientProfilePage.tsx, but I could be wrong.

**Comment by loafyyy** [comment]
I'm getting this error while testing the client profile page logged in as Gandalf_Wizard and viewing the client profile for Wormtongue
image

**Review by loafyyy — CHANGES_REQUESTED** [review]
I'm getting some errors testing the client profile page with the Gandalf_Wizard account while viewing the profile for Wormtongue. I'm not sure if it's a frontend or backend error.

**Comment by loafyyy** [comment]
I think the "Set profile pic" button shouldn't show up until the image is uploaded, otherwise having two buttons can be confusing.
image

File: src/components/AccountSecurity/MyOrganization.tsx (hunk 3 above)
Comment (crchong1) [reply src/components/AccountSecurity/MyOrganization.tsx:286]: theoretically logging is turned off on production so they cant actually see it but still we should remove all console log statements if possible.

**Review by crchong1 — APPROVED** [review]
awesome work melinda!

**Comment by loafyyy** [comment]
Why do we need this "password" field if we're not showing it?
image

**Comment by loafyyy** [comment]
For the Recent Activity, do we need the name on each row if it's the login activity for the same person? Similarly, do we need "LoginActivity" on each row? Can we just make the header "Recent Login Activity"? Also, what timezone is the time in (EST or PST or something else)? 
image

**Comment by loafyyy** [comment]
Can a client access their own profile page? Or is it only accessible through an Admin/Worker?

**Comment by loafyyy** [comment]
When I click "Edit Your Information", a modal for "Change Profile Photo" appears -- instead of having an intermediate step, can we just change the "Edit Your Information" to be the select profile photo button? Or are we also going to allow editing other client info like the address, email, etc.? Also, can a client access their own profile page? Or is it only accessible through an Admin/Worker? If it's the latter, why does the label say "Edit **Your** Information"?
image

**Comment by loafyyy** [comment]
Just a super small thing that I think would be nice and also easy to implement -- can the modal close when you click "Set Profile Photo"?

**Review by loafyyy — APPROVED** [review]
Good job @MelindaCardenas, the page functionality works for me! I just added some comments that you can address if you want (they should be addressed at some point), but overall I think the page is read to be merged into master.
